spec/requests/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'If response status was correct.' do
+    it 'If response status was correct for users' do
+      get users_path
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'If response status was correct.' do
+    it 'If response status was correct for users ID' do
+      get user_path(1)
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'Render correct template' do
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template('index')
+    end
+
+    it 'renders the show template' do
+      get user_path(1)
+      expect(response).to render_template('show')
+    end
+  end
+
+  describe 'Response body includes correct placeholder text' do
+    it 'displays the correct placeholder text' do
+      get users_path
+      expect(response.body).to include('Here is the list of Users')
+    end
+  end
+
+  describe 'Response body includes correct placeholder text' do
+    it 'displays the correct placeholder text' do
+      get user_path(1)
+      expect(response.body).to include('Here is it')
+    end
```

Comment:
- [ ] these and one of your test in posts_spec.rb is failing, kindly fix
![image](https://user-images.githubusercontent.com/66097371/225403473-a64f897a-ac74-4a2c-983c-8e201faa88ed.png)
![image](https://user-images.githubusercontent.com/66097371/225403575-ea64b492-0659-4fbe-9fd6-d22780527ccb.png)

Reply:
Alrighty